content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -51,7 +52,7 @@ do not already have a single-node cluster, you can create one by using
 [Persistent Volumes](/docs/concepts/storage/persistent-volumes/).
 -->
 
-* 你需要一个包含单个节点的 Kubernetes 集群，并且必须配置 kubectl 命令行工具以便与集群交互。
+* 你需要一个包含单个节点的 Kubernetes 集群，并且必须配置 {{< glossary_tooltip text="kubectl" term_id="kubectl" >}} 命令行工具以便与集群交互。
```

Comment:
Long sentences like this mixed Chinese and English, you can break it in the mixed spot, newlines will be parsed as spaces, like below:
```suggestion
* 你需要一个包含单个节点的 Kubernetes 集群，并且必须配置
  {{< glossary_tooltip text="kubectl" term_id="kubectl" >}} 命令行工具以便与集群交互。
```